### ISSUE TITLE
Question Generation

### ISSUE THREAD

**Issue by DivyaWadehra** [artitw/BERT_QA#1]
Does this solve question generation problem?

**Comment by artitw** [comment]
This is for evaluating the questions generated using SQuAD. However, if there is interest in the question generation part, let me know and I'll see what I can do.

**Comment by DivyaWadehra** [comment]
Yes, I want the code for question generation

**Comment by felicitywang** [comment]
@artitw Thanks for the great work. Is there any easy way to do inferencing and play with it(e.g. inferencing code + trained model or via an online demo etc.)?

**Comment by artitw** [comment]
@felicitywang @DivyaWadehra appreciate your interest in question generation very much. I will look into open-sourcing it as soon as I can. Currently it's not in an easily consumable form.

**Comment by DivyaWadehra** [comment]
Thankyou Looking forward for the code

**Comment by artitw** [comment]
@DivyaWadehra @felicitywang check out https://github.com/artitw/text2text

**Comment by felicitywang** [comment]
@artitw Thank you. However, this project is a pure wrapper of Microsoft's UniLM, which you didn't cite in your paper - is this UniLM the model you used in your paper?

**Comment by artitw** [comment]
Yes, that repo is based on UniLM. I think they released a good pretrained model we can easily play with. Ours is based on vanilla transformer; see https://github.com/artitw/tensor2tensor 
Currently I don't have the bandwidth to build an online demo for it. :(
